composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.8
+
+# Allow statements and log messages to immediately appear in the Cloud Run logs
+ENV PYTHONUNBUFFERED True
+
+COPY requirements.txt ./
```

Comment:
I'm not hot on docker files so this may be way off ... The requirements-composer.txt file include everything the requirements.txt file and then some. What's the difference between the two ... as you're coping both I guess we expect them to diverge at some point?

Reply:
Already answered in your blog:
To keep track of the PyPI modules installed in my Composer environment, I have been using a special `requirements-composer.txt` file that lives in the GitHub repository where I store my dags. Create this file now - it's just like a regular `requirements.txt` file, only with a special name. In it, I've added the most recent veersion of the operator package I'm using - in this case, `apache-airflow-backport-providers-google`